api/src/main/java/io/druid/timeline/TaskDataSegment.java

```diff
@@ -0,0 +1,94 @@
+package io.druid.timeline;
```

Comment:
@haoxiang47 there are many whitespace changes, can we please revert them?
https://github.com/druid-io/druid/pull/3831/files?w=1 indicates there's only new code added